cfgov/regulations3k/regdown.py

```diff
@@ -18,9 +18,8 @@ class RegulationsExtension(Extension):
 
     def extendMarkdown(self, md, md_globals):
         md.registerExtension(self)
-        md.parser.blockprocessors.add('labeledparagraph',
-                                      LabeledParagraphProcessor(md.parser),
-                                      '_begin')
+        md.parser.blockprocessors['paragraph'] = \
+            LabeledParagraphProcessor(md.parser)
```

Comment:
This change should fix the greediness of our `LabeledParagraphProcessor` by simply replacing the existing `paragraph` processor while maintaining its location in the `OrderedDict` that is `blockprocessors`.